docbook/reference/en/en-US/modules/Deprecated_security_modules.xml

```diff
@@ -0,0 +1,18 @@
+<chapter id="Deprecated_security_modules">
+    <title>Deprecated Security Modules</title>
+
+    <para>
+        These security modules are deprecated since 3.1.3.Final and users should use Keycloak provided by JBoss community instead:
+    </para>
+
+    <itemizedlist>
+        <listitem>keystone</listitem>
+        <listitem>login-module-authenticator</listitem>
+        <listitem>resteasy-oauth</listitem>
+        <listitem>skeleton-key-idm</listitem>
+    </itemizedlist>
+
+    <para>
+        The above modules are moved from 'security' directory to 'security-legacy' directory.
```

Comment:
How about something like the following line instead?

"Please refer to http://www.keycloak.org/ for a modern solution for the usecases previously covered by the deprecated modules".